apps/cloudfront-functions/src/__tests__/viewer-request-handler.test.ts

```diff
@@ -0,0 +1,45 @@
+import rewire from 'rewire';
+
+// rewire doesn't work with ts file. As workaround import the compiled js file
+const rewired = rewire('../../dist/viewer-request-handler');
+const handler = rewired.__get__('handler');
```

Comment:
Required when updating ESLint rules

```suggestion
// eslint-disable-next-line no-underscore-dangle
const handler = rewired.__get__('handler');
```